mobile/src/fdroid/java/be/mygod/vpnhotspot/util/FabricUtil.kt

```diff
@@ -0,0 +1,11 @@
+package be.mygod.vpnhotspot.util
+
+import android.content.Context
+
+object FabricUtil {
+
+    @Suppress("UNUSED_PARAMETER")
+    fun init(context: Context) {
+        //do nothing
+    }
+}
```

Comment:
You seem to disable logging in release builds. This is undesirable as those logs will be read upon exporting debug information.